src/components/transactions/transactionsOverview.js

```diff
@@ -93,14 +97,35 @@ class TransactionsOverview extends React.Component {
 
     return (
       <div className={`transactions ${styles.activity}`}>
-        <header>
-          <h2 className={styles.title}>{this.props.t('Activity')} {hasTitle
-            ? (<span>{this.props.t('of')} <span className={`${styles.accountTitle} account-title`}>{accountTitle}</span></span>)
-            : null}</h2>
-
+        <header className={`${grid.row}`}>
+          <h2 className={`${grid['col-xs-6']} ${grid['col-sd-6']} ${grid['col-md-9']} ${grid['col-lg-9']} ${styles.title}`}>
+          {this.props.t('Transaction')}
+          {
+            hasTitle && (<span>{this.props.t(' of')} <span className={`${styles.accountTitle} account-title`}>{accountTitle}</span></span>)
+          }
+          </h2>
+
+          {
+            this.props.match.url === Ulrs.wallet.path &&
+            (
+              <div className={`${grid['col-xs-5']} ${grid['col-sd-5']} ${grid['col-md-3']} ${grid['col-lg-3']} ${styles.headerButtons}`}>
+                <a className={'help-onboarding tx-receive-bt'}
+                  onClick={() => this.props.history.push(Ulrs.request.path) }>
+                  <FontIcon>request-token</FontIcon>
+                  {this.props.t('Receive')}
+                </a>
+                <ActionButton
+                  className={'tx-send-bt'}
+                  onClick={() => this.props.history.push(Ulrs.send.path) }
+                >
+                {this.props.t('Send')}
+                </ActionButton>
```

Comment:
Please use `<Link>` with `to` prop instead of `onclick` like in https://github.com/LiskHQ/lisk-hub/blob/959d542b412870e7fb435b604e978504b2cfb5a8/src/components/sendTo/send.js#L86-L90

Reply:
I made the change